tests/integration/deploy/test_deploy_command.py

```diff
@@ -675,6 +675,15 @@ def test_deploy_with_invalid_config(self, template_file, config_file):
         self.assertEqual(deploy_process_execute.process.returncode, 1)
         self.assertIn("Error reading configuration: Unexpected character", str(deploy_process_execute.stderr))
 
+    @parameterized.expand([("aws-serverless-function.yaml", "samconfig-tags-list.toml")])
```

Comment:
could you please add another test case to cover the old format (tags passed as one string)